eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -0,0 +1,44 @@
+param(
+  $RepoOwner,
+  $RepoName,
+  $BranchPrefix,
+  $AuthToken
+)
+
+. "${PSScriptRoot}\common.ps1"
+
+LogDebug "Operating on Repo [ $RepoName ]"
+try{
+  $branches = (List-References -RepoOwner $RepoOwner -RepoName $RepoName -Ref "heads/$BranchPrefix").ref
+}
+catch {
+  LogError "List-References failed with exception:`n$_"
+  exit 1
+}
+
+foreach ($branch in $branches)
+{
+  try {
+    $branchName = $branch.Replace("refs/heads/","")
+    $head = "${RepoOwner}/${RepoName}:${branchName}"
+    LogDebug "Operating on branch [ $branchName ]"
+    $pullRequests = List-PullRequests -RepoOwner $RepoOwner -RepoName $RepoName -head $head
+  }
+  catch
+  {
+    LogError "List-PullRequests failed with exception:`n$_"
+    exit 1
+  }
+
+  if ($pullRequests.Count -eq 0)
```

Comment:
Should we log cases where we find branches that have a pull request and list the branch and what open pull requests there are still for it?